lib/src/main/java/uk/gov/di/ipv/core/library/dto/ContraIndicatorEvidenceDto.java

```diff
@@ -13,5 +13,6 @@
 @AllArgsConstructor
 public class ContraIndicatorEvidenceDto {
     private String type;
-    private List<ContraIndicatorDto> contraIndicator;
+    private List<ContraIndicatorDto> ci;
```

Comment:
Is the attribute name changed from "contraIndicator" to "ci"?
As in https://github.com/alphagov/digital-identity-architecture/blob/main/rfc/0041-CI-mitigation-VC.md sample its still called "contraIndicator".

Reply:
Yes, it's changed. I believe the RFC will be updated to reflect the changes.